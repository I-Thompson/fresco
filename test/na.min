na-all.search
q
plot search-na-init.plot
min
migrad
end
show
plot search-na-fit.plot
q
exit
