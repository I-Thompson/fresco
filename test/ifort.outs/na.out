SFRESCO - FRES 3.4: Search Coupled Reaction Channels

 Please give search file name
 SFresco search file = na-all.search                                     

 Fresco input file = nalpha.nin                                        
   and output file = nalpha.frout                                      
 Search on  2 variables for   3 datasets,
 MNINIT UNABLE TO DETERMINE ARITHMETIC PRECISION. WILL ASSUME:  0.40E-06
  MINUIT RELEASE 96.03  INITIALIZED.   DIMENSIONS 100/ 50  EPSMAC=  0.80E-06

  Define SEARCH VARIABLES:

   Variable  1=V          is potential KP= 1, line= 2 col= 1
     value from Fresco input, step  0.0100, min,max   0.0000  0.0000

   Variable  2=Vso        is potential KP= 1, line= 3 col= 1
     value from Fresco input, step  0.0100, min,max   0.0000  0.0000

  Read definition of data set            1
  Read DATA SET  1 from file =                                                 
    of type  4           with absolute errors, CM  for state/excit  1  1,  idir= 0

  Excitation phase shift in channel  1.5- # 1
  Read data set            1
   Energy   Datum     Absolute error
    0.200   2.168      0.3000E-02
    0.400   7.606      0.8000E-02
    0.600   17.95      0.2000E-01
    0.800   36.04      0.3000E-01
    1.000   61.63      0.3000E-01
    1.200   85.44      0.4000E-01
    1.400   101.0      0.4000E-01
    1.600   110.1      0.4000E-01
    1.800   115.5      0.4000E-01
    2.000   118.8      0.4000E-01
    2.500   122.6      0.5000E-01
    3.000   123.6      0.5000E-01
    3.500   123.4      0.6000E-01
    4.000   122.8      0.8000E-01
    4.500   121.8      0.1000    
    5.000   120.8      0.1200    
    6.000   118.6      0.1600    
    7.000   116.3      0.2100    
    8.000   114.2      0.2600    
    9.000   112.2      0.3300    
   10.000   110.3      0.3900    
   12.000   106.9      0.5300    
   14.000   103.9      0.6700    
   16.000   101.2      0.8200    
   18.000   98.82      0.9600    
   20.000   96.57       1.100    
  Read definition of data set            2
  Read DATA SET  2 from file =                                                 
    of type  4           with absolute errors, CM  for state/excit  1  1,  idir= 0

  Excitation phase shift in channel  0.5- # 1
  Read data set            2
   Energy   Datum     Absolute error
    0.200  0.4280      0.3000E-02
    0.400   1.236      0.8000E-02
    0.600   2.314      0.1500E-01
    0.800   3.630      0.2000E-01
    1.000   5.150      0.3000E-01
    1.200   6.880      0.4000E-01
    1.400   8.780      0.5000E-01
    1.600   10.84      0.6000E-01
    1.800   13.04      0.7000E-01
    2.000   15.36      0.8000E-01
    2.500   21.51      0.1100    
    3.000   27.80      0.1300    
    3.500   33.84      0.1600    
    4.000   39.30      0.2000    
    4.500   44.04      0.2400    
    5.000   48.02      0.2700    
    6.000   53.90      0.3500    
    7.000   57.61      0.4200    
    8.000   59.82      0.4700    
    9.000   61.02      0.5100    
   10.000   61.56      0.5600    
   12.000   61.45      0.6500    
   14.000   60.53      0.7300    
   16.000   59.27      0.8300    
   18.000   57.90      0.9200    
   20.000   56.54       1.010    
  Read definition of data set            3
  Read DATA SET  3 from file =                                                 
    of type  4           with absolute errors, CM  for state/excit  1  1,  idir= 0

  Excitation phase shift in channel  0.5+ # 1
  Read data set            3
   Energy   Datum     Absolute error
    0.200  -10.88      0.2000E-01
    0.400  -15.37      0.3000E-01
    0.600  -18.80      0.4000E-01
    0.800  -21.69      0.4000E-01
    1.000  -24.23      0.4000E-01
    1.200  -26.52      0.4000E-01
    1.400  -28.62      0.5000E-01
    1.600  -30.57      0.5000E-01
    1.800  -32.39      0.5000E-01
    2.000  -34.11      0.5000E-01
    2.500  -38.04      0.5000E-01
    3.000  -41.57      0.5000E-01
    3.500  -44.78      0.5000E-01
    4.000  -47.74      0.6000E-01
    4.500  -50.50      0.6000E-01
    5.000  -53.08      0.7000E-01
    6.000  -57.79      0.9000E-01
    7.000  -62.01      0.1200    
    8.000  -65.82      0.1500    
    9.000  -69.29      0.1900    
   10.000  -72.44      0.2400    
   12.000  -77.90      0.3500    
   14.000  -82.34      0.5000    
   16.000  -85.81      0.7200    
   18.000  -88.31       1.060    
   20.000  -89.81       1.570    
 
  FRESCO output to nalpha.frout*
 
 Calculate scattering at energies
    0.200   0.400   0.600   0.800   1.000   1.200   1.400   1.600   1.800   2.000
    2.500   3.000   3.500   4.000   4.500   5.000   6.000   7.000   8.000   9.000
   10.000  12.000  14.000  16.000  18.000  20.000
 
    Var   1=V          value    43.000000
    Var   2=Vso        value    10.000000

  Total ChiSq/N = 26021.1113 from  76396.481   156.942  1509.911

 PARAMETER DEFINITIONS:
    NO.   NAME         VALUE      STEP SIZE      LIMITS
     1 'V         '    43.000      0.10000E-01     no limits
     2 'Vso       '    10.000      0.10000E-01     no limits
sfresco>     Var   1=V          value    43.000000, step   0.0100, error   0.0000
    Var   2=Vso        value    10.000000, step   0.0100, error   0.0000
sfresco> 
  ChiSq/N = 26021.111 from  76396.481   156.942  1509.911
  xmgr file written: search-na-init.plot                               
sfresco> 
   Call MINUIT
   with NOGradient, STRategy=0, ERRordef=  0.0128
 **********
 **    1 **SET NOGRADIENT 
 **********
 **********
 **    2 **SET STRAT    0.000    
 **********

 NOW USING STRATEGY  0: MINIMIZE THE NUMBER OF CALLS TO FUNCTION

 **********
 **    3 **SET ERRORDEF  0.1282E-01
 **********
minuit>  **********
 **    4 **MIGRAD 
 **********

 FIRST CALL TO USER FUNCTION AT NEW START POINT, WITH IFLAG=4.
 START MIGRAD MINIMIZATION.  STRATEGY 0.  CONVERGENCE WHEN EDM .LT. 0.13E-05

 FCN=   26021.11     FROM MIGRAD    STATUS=INITIATE       4 CALLS        5 TOTAL
                     EDM= unknown      STRATEGY= 0      NO ERROR MATRIX       

  EXT PARAMETER               CURRENT GUESS       STEP         FIRST   
  NO.   NAME        VALUE          ERROR          SIZE      DERIVATIVE 
   1     V          43.000       0.10000E-01   0.10000E-01   -42596.    
   2     Vso        10.000       0.10000E-01   0.10000E-01   -30633.    
                               ERR DEF= 0.128E-01
 MIGRAD FAILS TO FIND IMPROVEMENT
 MIGRAD FAILS WITH STRATEGY=0.   WILL TRY WITH STRATEGY=1.
 EIGENVALUES OF SECOND-DERIVATIVE MATRIX:
        -0.3746E+00  0.2375E+01
 MINUIT WARNING IN HESSE   
 ============== MATRIX FORCED POS-DEF BY ADDING  0.37697     TO DIAGONAL.

 FCN=   556.9422     FROM HESSE     STATUS=NOT POSDEF    14 CALLS       33 TOTAL
                     EDM=  0.21E+03    STRATEGY= 1      ERR MATRIX NOT POS-DEF

  EXT PARAMETER                APPROXIMATE        STEP         FIRST   
  NO.   NAME        VALUE          ERROR          SIZE      DERIVATIVE 
   1     V          43.427       0.89609E-02   0.62148       -251.69    
   2     Vso        10.823       0.12668E-01   0.15488        79.671    
                               ERR DEF= 0.128E-01
 MIGRAD FAILS TO FIND IMPROVEMENT
 MIGRAD TERMINATED WITHOUT CONVERGENCE.

 FCN=   394.2763     FROM MIGRAD    STATUS=FAILED        57 CALLS       58 TOTAL
                     EDM=  0.15E+05    STRATEGY= 1      ERR MATRIX NOT POS-DEF

  EXT PARAMETER                APPROXIMATE        STEP         FIRST   
  NO.   NAME        VALUE          ERROR          SIZE      DERIVATIVE 
   1     V          44.204       0.63248E-02    0.0000        3206.2    
   2     Vso        9.7255       0.89090E-02    0.0000       -237.08    
                               ERR DEF= 0.128E-01

 EXTERNAL ERROR MATRIX.    NDIM=  50    NPAR=  2    ERR DEF= 0.128E-01
  0.400E-04-0.562E-04
 -0.562E-04 0.794E-04
 ERR MATRIX NOT POS-DEF

 PARAMETER  CORRELATION COEFFICIENTS
       NO.  GLOBAL     1     2
        1  0.99652  1.000-0.997
        2  0.99652 -0.997 1.000
 ERR MATRIX NOT POS-DEF
minuit>  **********
 **    5 **END 
 **********

 CALL TO USER FUNCTION WITH IFLAG = 3

 END OF MINUIT COMMAND INPUT.    RETURN TO USER PROGRAM.

sfresco>  
  Dataset            1  <=>
   Energy   Datum      Abs. error  Theory         Chi
    0.200  2.1680     0.30000E-02  2.2788       1363.4608
    0.400  7.6060     0.80000E-02  7.9683       2051.1897
    0.600  17.950     0.20000E-01  18.684       1348.5462
    0.800  36.040     0.30000E-01  37.027       1082.8642
    1.000  61.630     0.30000E-01  62.065        210.3931
    1.200  85.440     0.40000E-01  84.820        240.3013
    1.400  101.00     0.40000E-01  99.750        976.0434
    1.600  110.11     0.40000E-01  108.60       1418.9108
    1.800  115.50     0.40000E-01  113.91       1582.6066
    2.000  118.79     0.40000E-01  117.19       1606.8894
    2.500  122.58     0.50000E-01  121.02        974.8371
    3.000  123.58     0.50000E-01  122.08        905.1246
    3.500  123.44     0.60000E-01  122.00        574.7707
    4.000  122.77     0.80000E-01  121.40        293.3350
    4.500  121.85     0.10000      120.53        172.9882
    5.000  120.80     0.12000      119.54        110.9339
    6.000  118.56     0.16000      117.38         54.0187
    7.000  116.32     0.21000      115.21         27.9385
    8.000  114.17     0.26000      113.11         16.7760
    9.000  112.15     0.33000      111.10         10.1209
   10.000  110.26     0.39000      109.20          7.3475
   12.000  106.86     0.53000      105.72          4.6279
   14.000  103.88     0.67000      102.61          3.5998
   16.000  101.23     0.82000      99.814          2.9839
   18.000  98.820     0.96000      97.284          2.5592
   20.000  96.570      1.1000      94.980          2.0905
 
  Dataset            2  <=>
   Energy   Datum      Abs. error  Theory         Chi
    0.200 0.42800     0.30000E-02 0.48450        354.6430
    0.400  1.2360     0.80000E-02  1.3909        374.9334
    0.600  2.3140     0.15000E-01  2.5898        338.1661
    0.800  3.6300     0.20000E-01  4.0345        409.1494
    1.000  5.1500     0.30000E-01  5.6944        329.2968
    1.200  6.8800     0.40000E-01  7.5436        275.2462
    1.400  8.7800     0.50000E-01  9.5573        241.7066
    1.600  10.840     0.60000E-01  11.710        210.1973
    1.800  13.040     0.70000E-01  13.974        178.1096
    2.000  15.360     0.80000E-01  16.322        144.5837
    2.500  21.510     0.11000      22.366         60.5214
    3.000  27.800     0.13000      28.316         15.7545
    3.500  33.840     0.16000      33.827          0.0062
    4.000  39.300     0.20000      38.691          9.2675
    4.500  44.040     0.24000      42.830         25.4091
    5.000  48.020     0.27000      46.261         42.4312
    6.000  53.900     0.35000      51.290         55.6147
    7.000  57.610     0.42000      54.450         56.5960
    8.000  59.820     0.47000      56.333         55.0536
    9.000  61.020     0.51000      57.356         51.6199
   10.000  61.560     0.56000      57.800         45.0920
   12.000  61.450     0.65000      57.628         34.5682
   14.000  60.530     0.73000      56.685         27.7473
   16.000  59.270     0.83000      55.377         22.0041
   18.000  57.900     0.92000      53.909         18.8198
   20.000  56.540      1.0100      52.389         16.8884
 
  Dataset            3  <=>
   Energy   Datum      Abs. error  Theory         Chi
    0.200 -10.880     0.20000E-01 -11.657       1509.0756
    0.400 -15.370     0.30000E-01 -16.409       1200.0137
    0.600 -18.800     0.40000E-01 -20.006        908.4225
    0.800 -21.690     0.40000E-01 -22.997       1067.2477
    1.000 -24.230     0.40000E-01 -25.597       1168.4291
    1.200 -26.520     0.40000E-01 -27.918       1221.4107
    1.400 -28.620     0.50000E-01 -30.025        789.4762
    1.600 -30.570     0.50000E-01 -31.961        774.4589
    1.800 -32.390     0.50000E-01 -33.758        748.5313
    2.000 -34.110     0.50000E-01 -35.437        703.9414
    2.500 -38.040     0.50000E-01 -39.219        556.1643
    3.000 -41.570     0.50000E-01 -42.542        377.6166
    3.500 -44.780     0.50000E-01 -45.513        215.0683
    4.000 -47.740     0.60000E-01 -48.206         60.3528
    4.500 -50.500     0.60000E-01 -50.671          8.1001
    5.000 -53.080     0.70000E-01 -52.944          3.7560
    6.000 -57.790     0.90000E-01 -57.025         72.2115
    7.000 -62.010     0.12000     -60.611        135.9462
    8.000 -65.820     0.15000     -63.808        179.9107
    9.000 -69.290     0.19000     -66.691        187.0425
   10.000 -72.440     0.24000     -69.316        169.4462
   12.000 -77.900     0.35000     -73.944        127.7707
   14.000 -82.340     0.50000     -77.927         77.9085
   16.000 -85.810     0.72000     -81.417         37.2277
   18.000 -88.310      1.0600     -84.519         12.7923
   20.000 -89.810      1.5700     -87.307          2.5425

  Total ChiSq/N =   394.2763 from    578.664   130.516   473.649
sfresco> 
  ChiSq/N =   394.276 from    578.664   130.516   473.649
  xmgr file written: search-na-fit.plot                                
sfresco>     Var   1=V          value    44.203831, step   0.0100, error   0.0063
    Var   2=Vso        value     9.725475, step   0.0100, error   0.0089
sfresco> 